documents/api/filtersets.py

```diff
@@ -0,0 +1,28 @@
+from django_filters import rest_framework as filters
+
+from ..models import Document
+
+
+class DocumentFilterSet(filters.FilterSet):
```

Comment:
Maybe at least the other or both (after and before) filters for dates should have either `lte` or `gte` as `lookup_expr` so that it would be possible to also include the document with the exact value.

Reply:
I thought about it as well, but my logic said that if it explicitly says `before/after`, it kinda implicitly assumes that the others are excluded, because then we would need to specify which boundary is inclusive and which isn't.

Reply:
Fair enough.